pom.xml

```diff
@@ -163,7 +163,7 @@
             <dependency>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-archetype-plugin</artifactId>
-                <version>2.4</version>
+                <version>3.0.0</version>
```

Comment:
On reflection, I wouldn't change this version right now. There is some brittle code elsewhere in the SDK, which is why the Travis build is failing.